docs/source/azure/policy/resources/vm.rst

```diff
@@ -0,0 +1,84 @@
+.. _azure_vm:
+
+Virtual Machines
+================
+
+Filters
+-------
+- Standard Value Filter (see :ref:`filters`)
```

Comment:
@stefangordon I didn't know if we also wanted to include things like the Generic Arm Filters (Metrics filter) here, so I left it out for now

Reply:
Sort of think that if we started listing every generic filter you could use we'd just have a huge list at the start of every resource page, we may just remove that value filter bit from them next time we work on this.